database/sql/init.sql

```diff
@@ -9,7 +9,10 @@ CREATE TABLE Users (
     is_providing_services BOOLEAN,
     location VARCHAR(255),
     location_coordinates_x DECIMAL(10, 6),
-    location_coordinates_y DECIMAL(10, 6)
+    location_coordinates_y DECIMAL(10, 6),
+    avatar VARCHAR(50),
```

Comment:
Tutaj widzę że avatar jednak powinien być intem, zgodnie z tym co mamy w RegistrationModelu. Początkowo myślałem że to będzie jakiś string, dlatego podałem że to ma być varchar. Sorki za zamieszanie

Reply:
Też nie mam co do tego pewności, ale raczej będą 4 awatary ustawione na sztywno. Poprawiłem